### DISCUSSION TITLE
Minha extensao parou de funcionar

### DISCUSSION THREAD

**Discussion by Mesq65** [aclap-dev/video-downloadhelper#714] [Help and Q&A]
### Your question

Minha extensao parou de funcionar nao consigo mais baixar videos e minha licença esta ok o que faço?

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://hotmart.com/pt-BR/club/medsize/products/1299981/content/97B6Pb0a7p?source=CLASS_TOP_ARROW",
  "error": {
    "details": "SideDownload error: : [in#1 @ 00000231cfcc0580] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://vod-akm.play.hotmart.com/video/MPq9AyJXqO/hls/MPq9AyJXqO-1676465296000-audio_2=128398.m3u8?hdntl=exp=1716914108~acl=/*~data=hdntl~hmac=7162c15714f0f84ff437c49ec48c9c1eabc8b154bacae4ab7653e776192a5845&app=f57b1bb2-30e3-4fd0-9c59-cfe00909f280&tokenParams=eyJhbGciOiJub25lIn0.eyJ0cmFuc2NvZGVkT2JqZWN0S2V5cyI6WyJ2aWRlby9NUHE5QXlKWHFPL21wNC84MjMzMTU1OC04ZWI3LTQyMTMtOTcxNC01Y2Y1ZWM1ZjAyODEubXA0IiwidmlkZW8vTVBxOUF5SlhxTy9tcDQvZWE1ZjkyMzgtZWJjYy00YTEyLTk1NzEtYzBkNmRjMmRiMGM3Lm1wNCIsInZpZGVvL01QcTlBeUpYcU8vbXA0LzIyM2VkMzMwLTJiNmQtNDVlNS04MWMzLWI0NDE2MWEzMDgyMi5tcDQiLCJ2aWRlby9NUHE5QXlKWHFPL21wNC8xMjVhYzVhNC0zOWM4LTRhMzMtYjM5Mi04MDFiNzlhMmVmZDgubXA0IiwidmlkZW8vTVBxOUF5SlhxTy9tcDQvMmMzNDc2NWItMTczYS00Y2VmLWFiYzAtZTJiYjQyNTlmODY4Lm1wNCJdLCJ1cGRhdGVEYXRlIjoxNjc2NDY1Mjk2MDAwLCJjZG5VcmwiOiJodHRwczovL3ZvZC1ha20ucGxheS5ob3RtYXJ0LmNvbSIsImRybSI6ZmFsc2UsInN1YiI6Ik1QcTlBeUpYcU8iLCJpYXQiOjE3MTI2NzMwNjR9..\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_1028732234502176",
    "error": "coapp_failure",
    "id": "coapp_failure_48858c54-b9e2-4439-8e23-c36033e50aae"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Over the years, many users have reported problems with Hotmart.  Also Udemy.  Those are both subscription web sites.  That means only subscribed users can even look at pages on the site.  So you're not going to get much help here.  You can search our old Google forum & find numerous such reports.  There was one user who said he didn't have any trouble, but it seems certain items on these sites are protected by DRM & not others.  If I recall correctly, the first lesson of a course was not protected by DRM so VDH could download it.  But the second & any subsequent lessons in the course were protected by DRM, which prevents VDH from downloading it.  Digital Rights Management is a copyright enforcement mechanism.  It's defined purpose is to prevent downloading of content.  The formula is:

DRM presence = VDH failure

So unless you know how to investigate problems further, you are out of luck.  If you want some clues about how to investigate problems further, search this forum for my mention of Plan B.  See if you can understand & use the techniques I use to investigate problems further.  Sometimes it works, sometimes it doesn't, but you can't know unless you try.

**Reply by petaqui** [reply]
But it is funny as with previous version it worked well. I have half of the course I bought downloaded, and with the new one, doesn't work

**Reply by Magna3** [reply]
Yes, my previous version it worked well too.

**Reply by GrampaWildWilly** [reply]
It is possible the web site changed how it offers material.  Maybe their older stuff works but their newer stuff doesn't.  You should verify whether the content you want is protected by DRM.  This is easy to do in Firefox but I don't use any other browser.  I have read comments on here that Chrome & Edge do not offer such an easy way to tell whether content is protected by DRM.  I can't advise you how to figure it out on those browsers.  All I know is what I've posted in this discussion.  I can only encourage you to try to apply my Plan B yourself & see if that tells you anything.
